cmd/minikube/cmd/node_add.go

```diff
@@ -57,6 +57,7 @@ var nodeAddCmd = &cobra.Command{
 
 		// Make sure to decrease the default amount of memory we use per VM if this is the first worker node
 		if len(cc.Nodes) == 1 && viper.GetString(memory) == "" {
+			warnAboutMultiNode()
```

Comment:
is this the only part of the code we could detect it is multi node? 
I think I saw somewhere else too

Reply:
the only time we care about warning the user is when they are starting a multinode cluster, so just `minikube start` or the first `minikube node add` which is what this is detecting

Reply:
so this condition " len(cc.Nodes) == 1 && viper.GetString(memory) == "" 

would detect both 
minikube node add
and also
minikube start -n=3?

wouldn;t the  len(cc.Nodes) be 3 in that case and not trigger this if?

Reply:
you're right that we shouldn't put the check inside this if because of the memory check. there'a separate call to `warnAboutMultiNode()` that will detect the `minikube start -n 3` case.

Reply:
fixed this check. the other check is in `start.go`